sippy-ng/src/releases/PayloadStreamOverview.js

```diff
@@ -0,0 +1,170 @@
+      })
+  }
+
+  useEffect(() => {
+    fetchData()
+  }, [])
+
+  if (fetchError !== '') {
+    return <Alert severity="error">Failed to load data, {fetchError}</Alert>
+  }
+
+  if (!isLoaded) {
+    return <p>Loading...</p>
+  }
+
+  return (
+    <Container>
+      <Grid item md={12}>
+        <Typography variant="h3" style={{ textAlign: 'center' }}>
+          Payload Stream Overview
+        </Typography>
+        <Grid container spacing={3} alignItems="stretch">
+          <Grid item md={3}>
+            <SummaryCard
+              key="payload-acceptance-summary-overall"
+              threshold={TEST_THRESHOLDS}
+              name="Overall"
+              success={streamHealth.phase_counts.total.accepted}
+              flakes={0}
+              caption={
+                <Fragment>
+                  <Tooltip
+                    title={`${streamHealth.phase_counts.total.accepted} runs`}
+                  >
+                    <span>
+                      {Number(
+                        (streamHealth.phase_counts.total.accepted /
+                          (streamHealth.phase_counts.total.accepted +
+                            streamHealth.phase_counts.total.rejected)) *
+                          100
+                      ).toFixed(2)}
+                      % Accepted
+                    </span>
+                  </Tooltip>
+                </Fragment>
+              }
+              fail={streamHealth.phase_counts.total.rejected}
+            />
+          </Grid>
+          <Grid item md={3}>
+            <SummaryCard
+              key="payload-acceptance-summary-currentweek"
+              threshold={TEST_THRESHOLDS}
+              name="This Week"
+              success={streamHealth.phase_counts.current_week.accepted}
+              flakes={0}
+              caption={
+                <Fragment>
+                  <Tooltip
+                    title={`${streamHealth.phase_counts.current_week.accepted} runs`}
+                  >
+                    <span>
+                      {Number(
+                        (streamHealth.phase_counts.current_week.accepted /
+                          (streamHealth.phase_counts.current_week.accepted +
+                            streamHealth.phase_counts.current_week.rejected)) *
+                          100
+                      ).toFixed(2)}
+                      % Accepted
+                    </span>
+                  </Tooltip>
+                </Fragment>
+              }
+              fail={streamHealth.phase_counts.current_week.rejected}
+            />
+          </Grid>
+          <Grid item md={6}>
+            <Typography variant="h6" style={{ textAlign: 'center' }}>
+              Last {streamHealth.count} payloads have been{' '}
+              {streamHealth.last_phase}
+            </Typography>
+            <Typography variant="h6" style={{ textAlign: 'center' }}>
+              Last payload accepted: {streamHealth.release_time}
```

Comment:
There's a relative time helper. Suggest:

```jsx
              <Tooltip title={streamHealth.release_time}>
                <Fragment>
                  Last payload accepted:{' '}
                  {relativeTime(new Date(streamHealth.release_time))}
                </Fragment>
              </Tooltip>
```

renders:

`Last payload accepted: 6 hours ago`

(with a tooltip showing the actual timestamp)